### PR TITLE
feat: Add example of very simple topology

### PR DESCRIPTION
**Pull request by vburckhardt**
### Description

https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/issues/199

**Check the relevant boxes:**
- [ ] Bug fix (nonbreaking change that fixes an issue)
- [ ] New feature (nonbreaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI related update (pipeline, etc.)

### Checklist

- [ ] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

File: examples/one-vpc-one-vsi/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+An example showing how the landing-zone module can be used to create a basic, minimal topology by using an `override.json` file:
+1. One single VPC:
+   - Named "management" in this example
+   - 3 subnets across the 3 availability zone to host VSIs
+   - Default, open network ACLs
+2. One single VSI:
+   - Named 'jump-box' in this example
+   - Located in one of the 'vsi' subnets of the VPC
+   - Publicly exposed via a floating IP.
+   - Open Security Groups
```
Comment (SirSpidey) [inline examples/one-vpc-one-vsi/README.md:14]:
```suggestion
1. A single VPC:
   - Named `management` in this example
   - Includes three subnets across the three availability zone to host VSIs
   - Default, open network ACLs
2. One single VSI:
   - Named 'jump-box' in this example
   - Located in one of the 'vsi' subnets of the VPC
   - Publicly exposed via a floating IP address.
   - Open security groups
```

File: examples/one-vpc-one-vsi/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+This example also create the minimum encryption and audit infrastructure:
+- Key protect instance and key used to encrypted the VSI boot volume
+- Activity Tracker infrastructure (activity tracker route to an encrypted COS bucket storing audit events)
```
Comment (SirSpidey) [inline examples/one-vpc-one-vsi/README.md:18]:
```suggestion
This example also creates the minimum encryption and audit infrastructure.
- A Key Protect instance and key that is used to encrypt the VSI boot volume
- The Activity Tracker infrastructure (Activity Tracker route to an encrypted COS bucket that stores audit events)
```

**Review by SirSpidey — CHANGES_REQUESTED** [review]
Editorial suggestions

**Review by SirSpidey — APPROVED** [review]
I think both of the PRs need the pre-commit hook run locally to update the readme with the example.

**Review by argeiger — DISMISSED** [review]

**Review by SirSpidey — APPROVED** [review]
👍

**Pull request merged**